site/content/docs/manifest/lb-web-service.en.md

```diff
@@ -229,6 +229,40 @@ List of all available properties for a `'Load Balanced Web Service'` manifest. T
             placement: 'private'
         ```
 
+    === "Multiple Ports Exposed"
+
+        ```yaml
+        name: 'frontend'
+        type: 'Load Balanced Web Service'
+
+        image:
+          build: './frontend/Dockerfile'
+          port: 8080
+
+        nlb:
+          port: 8080/tcp // exposed port 8080 of the main container i.e. frontend on the nlb port "8080/tcp"
+          additional_rules:
+            - port: 8084/tcp // exposed port 8084 of the main container i.e. frontend on the nlb port "8084/tcp
+            - port: 8085/tcp  // exposed port 3000 of the sidecar container i.e. envoy on the nlb port "8085/tcp
```

Comment:
In addition to the same comment suggestions from the other examples, we want to use `#` here 👀